### ISSUE TITLE
How many .com teams does GitHub have?

### ISSUE THREAD

**Issue by wadewilliams** [holman/feedback#597]
Hey-o, 

You had spoken in one of your talks about how you're exited about "getting back to working on the .com team".

 My hypothesis is that there are probably several .com teams working on all of the things at GitHub. Is that accurate? There's a lot of moving parts within the ".com" app, and I imagine jumping into improving a feature, like issues, probably has a smaller dedicated team within the ".com" team lattice. 

 For example again, GitHub Issues -- this is just one small part of the overall product, but how many people does the .com team dedicate to supporting this feature over time; and if the feature needs an improvement, (like the recent issues upgrade) - do you augment that team or just stick it in the hat of the team supporting it? Am I way off here? Any guidance here appreciated, please.

 In summary, I'm wondering about how many people you dedicate to supporting existing features. The ".com team" probably has end-to-end responsibility for a lot of different features -- and there's probably several sub-teams that are set up to accommodate support for the different areas of the product offering. 

 Could you expand on this? I can clarify the question further but I think this is a good starting point for the discussion. Thanks Holman.

[Edited for some additional clarity]
